app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,84 @@
+<h3>Recipe no.<%= @recipe.id %> full description</h3>
+<h3><%= @recipe.name %></h3>
+<section class="recipe-details">
+  <div class="d-flex gap">
+    <div class="right">
+      <p class="input">Preparation time: <%= @recipe.preparation_time_minutes %> mins </p>
+      <p  class="input">Cooking time: <%= @recipe.cooking_time_minutes %> mins </p>
+    </div>
+    <%= form_for @recipe, url: public_toggle_recipe_path(@recipe), method: :patch, remote: true, authenticity_token: true, html: { class: 'toggle-form' } do |f| %>
+        <div class="slider-container">
+        <span class="switch-label">Public</span>
+        <label class="switch">
+          <%= f.check_box :is_public, { class: "toggle-switch" }, 0, 1 %>
+          <span class="slider round"></span>
+        </label>
+      </div>
+      <%= f.hidden_field :is_public, value: @recipe.is_public ? 0 : 1 %>
+      <%= f.submit 'Toggle', class: 'hidden-submit', style: 'display: none;' %>
+    <% end %>
```

Comment:
- [ ] Great job on the project requirements but there's one small issue. On the recipe details page if I mark any recipe public or private I can see the toggle button changes but when I go to the public recipe page I can't see the changes there Also when I come back to the recipe details page again, I can only see recipe old state(public or private). Please make sure to fix this issue and changes to the recipe public state should reflect and users should be able to see the changes on the public recipe page.